package.json

```diff
@@ -4,23 +4,23 @@
     "@hotwired/turbo-rails": "^7.0.1",
     "@rails/actiontext": "^6.1.4-1",
     "@rails/ujs": "^6.1.4-1",
-    "@rails/webpacker": "5.4.0",
+    "@rails/webpacker": "^5.4.3",
     "@tailwindcss/forms": "^0.3.3",
     "autoprefixer": "^9",
     "core-js": "^3.8.0",
     "flatpickr": "^4.6.9",
-    "jquery": "3.5.1",
+    "jquery": "^3",
```

Comment:
Hebben we met de release van de tailwind versie niet alle jQuery eruit?

Reply:
Volgens mij niet. Anders waren we er [hier](https://github.com/DispuutHamers/webapp/blob/master/config/webpack/environment.js#L7) ook allang vanaf.

Reply:
Dat we hem in webpacker requiren betekend nog niet dat we hem ergens aanroepen, volgens mij doen we dat namelijk niet meer sinds de hele tailwind versie van de app.